packages/lab/src/content-status/ContentStatus.css

```diff
@@ -1,71 +1,39 @@
-/* Styles applied to root component (ContentStatus) */
 .uitkContentStatus {
   display: flex;
   flex-direction: column;
   justify-content: center;
   align-items: center;
-  height: 100%;
-  width: 100%;
-  letter-spacing: normal;
+  text-align: center;
+  gap: var(--uitk-size-unit);
 }
 
-/* Styles applied to inner content */
 .uitkContentStatus-content {
   color: var(--uitkContentStatus-content-color, var(--uitk-text-primary-foreground));
-  letter-spacing: var(--uitkContentStatus-letter-spacing, 0);
-  line-height: var(--uitkContentStatus-line-height, var(--uitk-text-base-lineHeight));
-  font-size: var(--uitkContentStatus-font-size, var(--uitk-text-fontSize));
-  min-height: var(--uitkContentStatus-min-height, var(--uitk-text-minHeight));
   display: flex;
   flex-direction: column;
-  justify-content: center;
-  align-items: center;
-  text-align: center;
-  width: 100%;
+  gap: var(--uitk-size-unit);
 }
 
-/* Styles applied to status title */
 .uitkContentStatus-title {
-  color: var(--uitkContentStatus-title-color, var(--uitk-text-primary-foreground));
-  letter-spacing: var(--uitkContentStatus-title-letter-spacing, 0);
-  font-weight: var(--uitkContentStatus-title-font-weight, var(--uitk-text-fontWeight-strong));
-  line-height: var(--uitkContentStatus-title-line-height, var(--uitk-text-base-lineHeight));
-  font-size: var(--uitkContentStatus-title-font-size, var(--uitk-text-fontSize));
-  min-height: var(--uitkContentStatus-title-min-height, var(--uitk-text-minHeight));
-  margin-top: var(--uitkContentStatus-title-margin-top, var(--uitk-size-unit));
-}
-
-/* Styles applied to status message */
-.uitkContentStatus-message {
-  margin-top: var(--uitkContentStatus-message-margin-top, var(--uitk-size-unit));
-  width: var(--uitkContentStatus-message-width, 100%);
+  font-weight: var(--uitk-typography-weight-semiBold);
 }
 
-/* Styles applied to status actions */
 .uitkContentStatus-actions {
-  margin-top: var(--uitkContentStatus-actions-margin-top, calc(2 * var(--uitk-size-unit)));
+  margin-top: var(--uitkContentStatus-actions-margin-top, var(--uitk-size-unit));
 }
 
-/* Styles applied if `status="error"` */
```

Comment:
why have the comments been deleted? I think these are used for the generated class table on docs page

Reply:
I find them a bit pointless. they're stating the obvious. Did I delete one that you find important? 
I don't see them in other components. Is this something we need to make sure we add to our css files?

Reply:
yes, if you go to 'CSS Classes' under the docs this is where the comments are used https://uitk.pages.dev/?path=/docs/documentation-core-button--button-variants

Reply:
added back in https://github.com/jpmorganchase/uitk/pull/260